semgrep_output_v0.ts

```diff
@@ -35,6 +35,12 @@ export type CoreMatch = {
 export type CoreMatchExtra = {
   message?: string;
   metavars: Metavars;
+  dataflow_trace?: DataflowTrace;
+}
+
+export type DataflowTrace = {
+  taint_source?: Location[];
```

Comment:
Are the locations guaranteed to be in order of evaluation?

Reply:
For now we don't plan on making such guarantees, unless it provides concrete benefits to "consumers".